pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -610,17 +616,43 @@ func (r *reconcilerBase) validateCACertSecret(ctx context.Context, namespace, ca
 			return errors.Wrapf(err, "Secret %s get failed", caCertSecretRefName)
 		}
 		if _, ok := secret.Data[CACertSecretKey]; !ok {
-			return fmt.Errorf("caCertSecretRef was set, but %s key is not present in %s Secret", CACertSecretKey, caCertSecretRefName)
+			return errors.Errorf("caCertSecretRef was set, but %s key is not present in %s Secret", CACertSecretKey, caCertSecretRefName)
 		}
 	}
 	return nil
 }
 
+func (r *reconcilerBase) validateAnnotations(_ context.Context, rs client.Object) error {
+	autoscalingStrategy := reconcilerAutoscalingStrategy(rs)
+	if autoscalingStrategy != metadata.ReconcilerAutoscalingStrategyAuto &&
```

Comment:
Probably would be cleaner with a `switch`

Reply:
I'm not sure it's better, but i changed it. lmk.